src/main/java/jenkins/plugins/gerrit/workflow/GerritReviewStep.java

```diff
@@ -64,10 +68,27 @@ protected Void run() throws Exception {
           int changeId = Integer.parseInt(matcher.group("changeId"));
           int revision = Integer.parseInt(matcher.group("revision"));
           listener.getLogger().format("Gerrit review change %d/%d label %s=%d (%s)%n", changeId, revision, label, score, message);
-          ReviewInput reviewInput = new ReviewInput().label(label, score).message(message);
+          ReviewInput reviewInput = new ReviewInput().message(message);
+          boolean notifyOwner = false;
+          if (labels == null) {
```

Comment:
Why don't you do something different and internal use always labels?
From an external point of view, just add an extra pair based on the legacy `label` and `score` variables.

Reply:
because I would like to remove the legacy in next patch :)
had it been gerrit I had already submitted this for review... but as this github is branch based I cannot do this without all-or-nothing.